lua/hbac/init.lua

```diff
@@ -92,7 +92,38 @@ local function close_unused()
 	end
 end
 
+
+-- Create commands to add and remove buffers from the persistant_buffers table
+M.release_buffer = function(bufnr)
+	bufnr = bufnr or vim.api.nvim_get_current_buf()
+	M.persistant_buffers[bufnr] = nil
+end
+
+M.commands = {
+	pin = M.persist_buffer,
+	unpin = M.release_buffer,
+	close_unused = close_unused,
+}
+
+function M.hbac_cmd(arg)
+	if M.commands[arg] then
+		M.commands[arg]()
+		print("Hbac: Buffer " .. arg .. "ned")
```

Comment:
```suggestion
	if arg == "close_unused" then
		close_unused()
		print("Hbac: Unused buffers closed")
	elseif M.commands[arg] then
		M.commands[arg]()
		print("Hbac: Buffer " .. arg .. "ned")
```

🤦  How about this? And are you ok with `Hbac` vs. `hbac` in this message? For the command itself, I was following the capitalization convention.

Reply:
Hbac makes sense, since all user commands in vim have to start with a capital letter